docs/pages/repo/docs/reference/configuration.mdx

```diff
@@ -292,3 +295,23 @@ Set type of output logging.
   }
 }
 ```
+
+### `persistent`
+
+`type: boolean`
+
+Label a task as `persistent` if it is a long-running process, such as a dev server or `--watch` mode.
+Turbo will prevent other tasks from depending on persistent tasks.
```

Comment:
I imagine we'll get an issue to support the persistent tasks depending on each other. If we have an existing issue explaining why we aren't doing this linking to that here could be good. If we don't then we can just wait for a new issue :)

Reply:
I think we need to stay open-ended on that right now. I can think of two ways of supporting it:

- in collaboration with turbopack's file watching (probably quite far down the line)
- some kind of "hoisted parallel", where if we see a task depending on a persistent task, we detach part of the graph and make it parallel.

The general feedback from @mattpocock was that no-one is asking for persistent task dependencies, so we don't need to solve for it right now.